### PR TITLE
#1633 Typos

### LINKED ISSUE #1633: Fix typos
During development I found a few typos I'd like to correct here.

### PR DESCRIPTION
**Pull request by jhorcicka**
Fixes #1633 

Typos fixes.

### CODE REVIEW

File: engine/core/src/main/java/org/datacleaner/job/runner/JobStatus.java [hunk 1]
```diff
@@ -21,5 +21,5 @@
 
 public enum JobStatus {
 
-    SUCCESSFUL, ERRORNOUS, NOT_FINISHED
+    SUCCESSFUL, ERRONEOUS, NOT_FINISHED
```
Comment (LosD) [inline engine/core/src/main/java/org/datacleaner/job/runner/JobStatus.java]: Part of the public API, so unfortunately not going to fly.

File: engine/core/src/main/java/org/datacleaner/job/runner/AnalysisResultFutureImpl.java [hunk 2]
```diff
@@ -152,7 +152,7 @@ public boolean isSuccessful() {
     }
 
     @Override
-    public boolean isErrornous() {
+    public boolean isErroneous() {
```
Comment (LosD) [inline engine/core/src/main/java/org/datacleaner/job/runner/AnalysisResultFutureImpl.java]: Part of the external API, so unfortunately not going to fly.

Comment (LosD) [inline engine/core/src/main/java/org/datacleaner/job/runner/AnalysisResultFutureImpl.java]: (That of course also goes for the interface this implements... To lazy to hunt for it :))

**Comment by ClaudiaPHI** [comment]
LGTM!

**Comment by ClaudiaPHI** [comment]
Dennis is right.

**Comment by jhorcicka** [comment]
Updated, @LosD, @ClaudiaPHI.

File: engine/core/src/main/java/org/datacleaner/job/runner/AnalysisResultFuture.java [hunk 3]
```diff
@@ -76,45 +76,45 @@
...
+     * Finds (and waits if necessary) the results of a single Analyzer.
      *
      * @param componentJob
      *            the component job (typically AnalyzerJob) to find the result
      *            for
      * @return the result for a given component job
      * @throws AnalysisJobFailedException
      *             if the analysis did not go well (use isSuccessful() or
-     *             isErrornous() to check)
+     *             isErroneous() to check)
```
Comment (LosD) [inline engine/core/src/main/java/org/datacleaner/job/runner/AnalysisResultFuture.java]: This is now incorrect

File: engine/core/src/main/java/org/datacleaner/job/runner/AnalysisResultFuture.java [hunk 4]
```diff
@@ -76,45 +76,45 @@
...
     /**
-     * Finds (and waits if nescesary) the results mapped to the Analyzer jobs
+     * Finds (and waits if necessary) the results mapped to the Analyzer jobs
      *
      * @return a map with ComponentJobs as keys to the corresponding
      *         AnalyzerResults.
      * @throws AnalysisJobFailedException
      *             if the analysis did not go well (use isSuccessful() or
-     *             isErrornous() to check)
+     *             isErroneous() to check)
```
Comment (LosD) [inline engine/core/src/main/java/org/datacleaner/job/runner/AnalysisResultFuture.java]: This is now incorrect

**Comment by LosD** [comment]
A few ones where the comment doesn't fit the method anymore, otherwise LGTM

**Comment by LosD** [comment]
LGTM (sorry to have reduced the amount of improvements possible, though)!

**Comment by jhorcicka** [comment]
Better something than nothing! Thanks.

**Pull request merged**